test/e2e/validation_test.go

```diff
@@ -254,3 +262,41 @@ func filterSSMatrix(mat *types.ComMatrix) (*types.ComMatrix, error) {
 
 	return &types.ComMatrix{Matrix: res}, nil
 }
+
+func extractNotUsedAndMissingEPS(csvContent string) (*types.ComMatrix, *types.ComMatrix, error) {
```

Comment:
can we split this into 2 clear functions? not sure I see the benefit of combining them, would be cleaner having one function for unused and one for missing returning a matrix, and let the caller call w.e they want